doc/cephfs/experimental-features.rst

```diff
@@ -60,6 +60,13 @@ writing, but there is insufficient testing to provide stability guarantees and
 every expansion of testing has generally revealed new issues. If you do enable
 snapshots and experience failure, manual intervention will be needed.
 
+Snapshots are known not to work properly with multiple filesystems (below) in
+some cases. Specifically, if you share a pool for multiple FSes and delete
```

Comment:
You mean specifically sharing data pools, right?  I think Doug has a patch to forbid that (it's definitely already unsafe).

Then pool sharing will become possible with, http://tracker.ceph.com/issues/15066 so I guess at that point we'll need to put something in to ensure that at most one filesystem using a particular data pool can have snapshots enabled.